src/Components/SeatManager/SeatManager.js

```diff
@@ -88,7 +91,8 @@ class SeatManager extends Component {
             </div>
 
             <button className='send-order-button'
-              onClick={() => this.sendOrder()}>Send Order</button>
+              onClick={() => this.sendOrder()}
+              disabled={this.props.currentTableOrder.length <= 0}>Send Order</button>
```

Comment:
Can you use !this.props.currentTableOrder.length? I

Reply:
totally. just made that change. thanks, @lfinney!